plugin-dev/Scripts/upload-debug-symbols-win.bat

```diff
@@ -78,6 +78,8 @@ if "%TargetPlatform%"=="Win64" (
     set CliExec=%PluginPath:"=%\Source\ThirdParty\CLI\sentry-cli-Windows-x86_64.exe
 ) else if "%TargetPlatform%"=="LinuxArm64" (
     set CliExec=%PluginPath:"=%\Source\ThirdParty\CLI\sentry-cli-Windows-x86_64.exe
+) else if "%TargetPlatform%"=="XSX" (
```

Comment:
Can we mention `XSX` platform in public repo?

Reply:
I've removed this check entirely as we should now be able to upload symbols for all platforms Unreal allows us to package for on Windows.